tests/test_resources.py

```diff
@@ -23,39 +23,136 @@
     mongodb_fixture,
 )
 
-from mass.adapters.outbound.dao import DaoNotFoundError
+from mass.adapters.outbound.aggregator import AggregatorNotFoundError
 from mass.core import models
 from tests.fixtures.joint import JointFixture, joint_fixture  # noqa: F401
 from tests.fixtures.mongo import populated_mongodb_fixture  # noqa: F401
 
 
 @pytest.mark.asyncio
-async def test_basic_query_and_load(joint_fixture: JointFixture):  # noqa: F811
-    """Make sure we can pull back the documents that are stored as well as load more"""
+async def test_basic_query(joint_fixture: JointFixture):  # noqa: F811
+    """Make sure we can pull back the documents as expected"""
 
+    # pull back all 3 test documents
```

Comment:
Since these different test senarios don't build on each other, it would be better to put them into different test functions.

Reply:
I see your point, will change that.